internal/controller/nodeagent.go

```diff
@@ -37,7 +35,6 @@ const (
 	IBMCloudPVHostPath      = "/var/data/kubelet/pods"
 	GenericPluginsHostPath  = "/var/lib/kubelet/plugins"
 	IBMCloudPluginsHostPath = "/var/data/kubelet/plugins"
-	ResticPVHostPathEnvVar  = "RESTIC_PV_HOSTPATH"
```

Comment:
I think you will also need to remove `RESTIC_PV_HOSTPATH` from manager.yaml and operator csv

Reply:
good catch, will update